Resources/Computing/ComputingElement.py

```diff
@@ -472,7 +472,7 @@ def getResourceDict( ceName = None ):
   # now add some defaults
   resourceDict['Setup'] = gConfig.getValue( '/DIRAC/Setup', 'None' )
   if not 'CPUTime' in resourceDict:
-    resourceDict['CPUTime'] = maxCPUSegments[-1]
+    resourceDict['CPUTime'] = gConfig.getValue( '/LocalSite/CPUTime' )
```

Comment:
I personally find it debatable the need for a hardcoded default. 
But I agree that for a patch it's better to play it safe, so I'll push this change in a second.